apps/namadillo/src/App/Governance/Governance.tsx

```diff
@@ -1,41 +1,26 @@
-import { useSanitizedLocation } from "@namada/hooks";
-import { useEffect } from "react";
-import { Route, Routes, useNavigate } from "react-router-dom";
+import { Route, Routes } from "react-router-dom";
 import { GovernanceOverview } from "./GovernanceOverview";
 import { ProposalAndVote } from "./ProposalAndVote";
 import { SubmitVote } from "./SubmitVote";
 import { ViewJson } from "./ViewJson";
 import GovernanceRoutes from "./routes";
 
-export const Governance: React.FC = () => {
-  const navigate = useNavigate();
-  const location = useSanitizedLocation();
-
-  // from outside this view we just navigate here
-  // this view decides what is the default view
-  useEffect(() => {
-    if (location.pathname === GovernanceRoutes.index()) {
-      navigate(GovernanceRoutes.overview().url);
-    }
-  }, [location.pathname]);
-
-  return (
-    <main className="w-full">
-      <Routes>
-        <Route
-          path={`${GovernanceRoutes.overview()}`}
-          element={<GovernanceOverview />}
-        />
-        <Route
-          path={`${GovernanceRoutes.proposal()}`}
-          element={<ProposalAndVote />}
-        />
-        <Route
-          path={`${GovernanceRoutes.submitVote()}`}
-          element={<SubmitVote />}
-        />
-        <Route path={`${GovernanceRoutes.viewJson()}`} element={<ViewJson />} />
-      </Routes>
-    </main>
-  );
-};
+export const Governance: React.FC = () => (
+  <main className="w-full">
+    <Routes>
+      <Route
+        path={`${GovernanceRoutes.overview()}`}
```

Comment:
We can also make this Route with the property `index`

Reply:
`index` didn't seem to work (I think because of a missing trailing `/`), so I just used `"/*"` to match the Staking routes.